src/macsniff.cpp

```diff
@@ -67,13 +68,10 @@ bool mac_add(uint8_t *paddr, int8_t rssi, bool sniff_type) {
             rgb_set_color(COLOR_NONE);
         }
      
-        if (added) { // first time seen this WIFI or BLE MAC
-            ESP_LOGI(TAG, "%s RSSI %ddBi -> Hash %04X -> WiFi:%d  BLE:%d  %s", 
-                                typebuff, rssi, hashedmac, 
-                                (int) wifis.size(), (int) bles.size(), buff );
-        } else { // already seen WIFI or BLE MAC
-            ESP_LOGI(TAG, "%s RSSI %ddBi -> Hash %04X -> already seen", typebuff, rssi, hashedmac);
-        }
+        ESP_LOGI(TAG, "%s RSSI %ddBi -> MAC %s -> Hash %04X -> WiFi:%d  BLE:%d  %s", 
```

Comment:
It seems this logic is not working properly, already seens are reported as new:


```
[I][main.cpp:272] sniffer_loop(): Wifi set channel 5
[I][main.cpp:272] sniffer_loop(): Wifi set channel 6
[I][macsniff.cpp:74] mac_add(): WiFi RSSI -88dBi -> MAC E510B339 -> Hash E855 -> WiFi:2  BLE:12  already seen
[I][macsniff.cpp:74] mac_add(): WiFi RSSI -87dBi -> MAC E510B339 -> Hash E855 -> WiFi:2  BLE:12  new
[I][main.cpp:272] sniffer_loop(): Wifi set channel 7
[I][macsniff.cpp:74] mac_add(): WiFi RSSI -84dBi -> MAC 4F4284B6 -> Hash BCD0 -> WiFi:3  BLE:12  already seen
[I][macsniff.cpp:74] mac_add(): WiFi RSSI -87dBi -> MAC 4F4284B6 -> Hash BCD0 -> WiFi:3  BLE:12  new
[I][main.cpp:272] sniffer_loop(): Wifi set channel 8
[I][macsniff.cpp:74] mac_add(): WiFi RSSI -82dBi -> MAC 37BB0FB6 -> Hash DCF1 -> WiFi:4  BLE:12  already seen
[I][macsniff.cpp:74] mac_add(): WiFi RSSI -86dBi -> MAC 37BB0FB6 -> Hash DCF1 -> WiFi:4  BLE:12  new
[I][main.cpp:272] sniffer_loop(): Wifi set channel 9
[I][main.cpp:272] sniffer_loop(): Wifi set channel 10
```
[

Reply:
callback are called async and serial display can be reversed order of what has really been done with multi tasking (already seen this). so as you can see device is counted only once, it would be relevant if it was already seen before new on next same channel scan.